docs/java/features/connectivity/destinations.md

```diff
@@ -29,17 +29,38 @@ DestinationAccessor.getDestination("my-destination");
 This will look up the destination in the destination service if the application is running on SAP Cloud Platform. But
 also, other sources like the environment variables are considered.
 
+## Decorating Destinations ##
+Depending on the use case, one needs to wrap the accessed destination before making a request to any system. This is to make sure all required destination properties are correctly set before invoking the actual request. 
```

Comment:
```suggestion
Depending on the use case, one needs to wrap the accessed destination before issuing a request to a system. This is to make sure all required destination properties are correctly set before invoking the actual request.
```

Reply:
Just to make it not sound like you need to decorate to make "any request at all"

Reply:
Good point:)